rust/pact_mock_server_ffi/src/lib.rs

```diff
@@ -111,6 +108,56 @@ pub unsafe extern fn init(log_env_var: *const c_char) {
   builder.try_init().unwrap_or(());
 }
 
+/// Sets the log level explicitly
+///
+/// # Safety
+///
+/// Exported functions are inherently unsafe.
+#[no_mangle]
+pub unsafe extern fn log_level(level: *const c_char) {
```

Comment:
Maybe re-name this, as it initialises the logging, which can only be done once. I.e., someone might think they can call it again later to change the log level, and that won't do anything.